docs/src/components/Layout.js

```diff
@@ -36,11 +36,6 @@ import axios from 'axios';
 const useGetStorybookData = async (name) => {
   let componentName = name;
 
-  if (!name.startsWith('components')) {
```

Comment:
why we removed this?

Reply:
There are some stories that start with the "patterns", "library", etc. keywords.  This condition prepends "components" at the start of the name which will break that particular story.